out/manifest.go

```diff
@@ -24,18 +24,34 @@ func NewManifest(manifestPath string) (Manifest, error) {
 	return manifest, nil
 }
 
-func (manifest *Manifest) EnvironmentVariables() map[interface{}]interface{} {
-	envVars, hasEnvVars := manifest.data["env"].(map[interface{}]interface{})
```

Comment:
Hmm, should we still respect the `env` if it's there?

Reply:
We don't modify the top-level `env` config if it's set, so if I'm reading your question correctly, we do respect it. We're only adding/modifying nested `env` keys now, so any values set in the top-level `env` won't get changed.

Reply:
Ah, ok, just noticed this is all in service of writing the manifest to a file. I thought this method would be used for submitting the app with the returned env vars, but it looks like it's for mutating.